ocs_ci/ocs/utils.py

```diff
@@ -799,6 +800,22 @@ def collect_ocs_logs(dir_name, ocp=True, ocs=True):
             ocp_log_dir_path, ocp_must_gather_image,
             '/usr/bin/gather_service_logs worker'
         )
+    if mcg:
+        from ocs_ci.ocs.resources.pod import get_pods_having_label, download_file_from_pod, Pod
+        nb_db_pod = Pod(
+            **get_pods_having_label(
+                label=constants.NOOBAA_DB_LABEL,
+                namespace=defaults.ROOK_CLUSTER_NAMESPACE
+            )[0]
+        )
+        ocs_log_dir_path = os.path.join(log_dir_path, 'noobaa_db_dump')
+        create_directory_path(ocs_log_dir_path)
+        ocs_log_dir_path = os.path.join(ocs_log_dir_path, 'nbcore.gz')
+        nb_db_pod.exec_cmd_on_pod("mongodump --archive=nbcore.gz --gzip --db=nbcore")
+        download_file_from_pod(
+            pod_name=nb_db_pod.name, remotepath="nbcore.gz",
```

Comment:
Not sure if that would download the `nbcore.gz`
I think that you should add the working directory of the container prior to that file.
If I'm not mistaken `/opt/app-root/src` is the path that needs to be added.
Example of the command that should work: `oc cp -n openshift-storage noobaa-db-0:/opt/app-root/src/nbcore.gz ./nbcore.gz`

Reply:
This works as is. Should I still change according to your suggestion?

Reply:
I will change to `oc cp -n openshift-storage noobaa-db-0:/opt/app-root/src/nbcore.gz ./nbcore.gz`

Reply:
done

Reply:
I'm not sure how did it work for you since I'm connected to a cluster and I run the following commands:
```
oc exec -n openshift-storage noobaa-db-0 -- mongodump --archive=nbcore.gz --gzip --db=nbcore
```
```
oc cp -n openshift-storage noobaa-db-0:/nbcore.gz ./nbcore.gz             
tar: Removing leading `/' from member names
tar: /nbcore.gz: Cannot stat: No such file or directory
tar: Exiting with failure status due to previous errors
error: nbcore.gz no such file or directory
```
And it fails.
But when I run:
```
oc cp -n openshift-storage noobaa-db-0:/opt/app-root/src/nbcore.gz ./nbcore.gz              

tar: Removing leading `/' from member names
```
It doesn't fail and copies the dump.

Reply:
this is the command it runs:
`oc -n openshift-storage cp noobaa-db-0:/opt/app-root/src/nbcore.gz /tmp/failed_testcase_ocs_logs_1592314638/test_bucket_creation[3-S3]_ocs_logs/noobaa_db_dump/nbcore.gz`

Reply:
Yeah, after your fix to my comment. This should work fine.
Previously it ran:
```
oc -n openshift-storage cp noobaa-db-0:/nbcore.gz /tmp/failed_testcase_ocs_logs_1592314638/test_bucket_creation[3-S3]_ocs_logs/noobaa_db_dump/nbcore.gz
```
Unless I understood something incorrectly from the code.